docs/rst/notes/forthcoming_version.rst

```diff
@@ -1,7 +1,11 @@
 .. add orphan tag when new info added to this file
 
-:orphan:
+.. :orphan:
 
 ###################
 Forthcoming Version
 ###################
+
+This release includes the following **Recording features**:
+
+* :ref:`Resource Limits <recorder_specs_logging>`.
```

Comment:
Some people (I'm not one of those) prefer to have longer entries, with a verb at least.

Reply:
I'm not one of them either.